### PR TITLE
Schemaless

### PR DESCRIPTION
**Pull request by bidoubiwa**
Removed name,
Added identifier
removed Schema

### CODE REVIEW

File: guides/main_concepts/documents.md [hunk 1]
```diff
@@ -36,9 +35,17 @@ curl -X POST `http://localhost:7700/indexes/movie/documents` \
...
 A **document must contain** [one identifier field](/guides/main_concepts/documents.md#identifier) to be indexed in MeiliSearch.
 
+## Fields
+
+All fields present in a document are automaticly <glossary word="searchable" /> and <glossary word="displayed" />.
+The way MeiliSearch handles a field is customizable in the settings <Badge text="soon" type="warn"/>. You can make a field only searchable, or only displayed, or none, in this case MeiliSearch will completely ignore the field when it is send on document addition.
+
+You can also add <glossary word="ranking rules" /> on a field, to, for example, add a rule that makes recent movies more relevant than older ones.
+
+
```
Comment (curquiza) [inline guides/main_concepts/documents.md]:
```suggestion
```

File: guides/main_concepts/documents.md [hunk 2]
```diff
@@ -65,17 +72,21 @@ If the identifier is not found in a document, it will not be added.
 
 MeiliSearch has several ways of knowing which field is the `identifier`.
 
-- MeiliSearch [finds the id](/guides/main_concepts/documents.md#finding-the-identifier) based on your first document.
-- You set it [on index creation](/guides/main_concepts/documents.md#setting-the-identifier) <Badge text="soon" type="warn"/>
+- MeiliSearch [infers automaticly the identifier](/guides/main_concepts/documents.md#meilisearch-infers-your-identifier) based on your first document.
```
Comment (curquiza) [inline guides/main_concepts/documents.md]:
```suggestion
- MeiliSearch [automatically infers the identifier](/guides/main_concepts/documents.md#meilisearch-infers-your-identifier) based on your first document.
```

File: guides/main_concepts/documents.md [hunk 3]
```diff
@@ -65,17 +72,21 @@ If the identifier is not found in a document, it will not be added.
...
+#### MeiliSearch infers your identifier
 
 If no identifier has been given through the index creation or through document additions, MeiliSearch will search for the identifier field in the first document sent.
 
 MeiliSearch will search for an attribute that contains the string `id` in any way case-insensitively. (i.e, `uid`, `MovieId`, `ID`, `123id123`).
 If none has been found, no documents will be added.
 
+#### Missing identifier error
+
+❗️ If you have the `missing identifier` error, it means MeiliSearch could not recognize your identifier. This mean your identifier does not answer the formatting explained above. To solve this error, you could send the [identifier's name as a query parameter](/references/documents.md#add-or-replace-documents) or [update your index  to add the identifier's name](/references/indexes.md#create-an-index) as explained in: [setting the identifier](/guides/main_concepts/documents.md#identifier).
```
Comment (curquiza) [inline guides/main_concepts/documents.md]:
```suggestion
❗️ If you get the `missing identifier` error, it means MeiliSearch could not recognize your identifier. This means your identifier does not answer the formatting explained above. To solve this error, you could:
- send the [identifier's name as a query parameter](/references/documents.md#add-or-replace-documents)
- or [update your index to add the identifier's name](/references/indexes.md#create-an-index).
Theses options are explained in: [setting the identifier](/guides/main_concepts/documents.md#identifier).
```

Comment (curquiza) [reply guides/main_concepts/documents.md]: (I split because the sentence was really long)

File: guides/main_concepts/indexes.md [hunk 4]
```diff
@@ -1,152 +1,81 @@
 # Indexes
 
-An `index` is an entity, like a *table in SQL*, with a specific `schema` definition. It gathers a collection of `documents` with the structure defined by the schema.
+An index is the collection of a certain type of data.
 
-An **index is defined** by an unique identifier `uid` that is generated by MeiliSearch (*if none is given*) on [index creation](/references/indexes.md#create-an-index). It also has **a name** to help you track your different indexes.
+It is, like a table in `SQL`, or a collection in `mongoDb`, an entity that collects a set of documents.
```
Comment (curquiza) [inline guides/main_concepts/indexes.md]:
```suggestion
It is, like a table in `SQL`, or a collection in `MongoDB`, an entity that collects a set of documents.
```

File: guides/main_concepts/indexes.md [hunk 5]
```diff
@@ -1,152 +1,81 @@
...
 
-## Index UID and name
+An index is defined by an `uid` and contains the following information:
+- One document identifier <glossary word="attribute"/>
+- A set of relevancy rules (based on presets and customization)
+- A list of synonyms and stop-words
+- Rules for each field of a document
+- Settings
```
Comment (curquiza) [inline guides/main_concepts/indexes.md]: For me the synonyms and stop-words were in settings, no?

File: guides/main_concepts/indexes.md [hunk 6]
```diff
@@ -1,152 +1,81 @@
...
-### Fields order
-
-The **order of the fields represents their relevance** in the search engine.
-
-Thus, if a `title` field is defined before a `description` field, its content will be considered more relevant to a search query than that of a "description" field.
+Each of the indexes has information about the fields found in the documents, how MeiliSearch handles them, and their order of importance. An index also has its own synonyms, relevancy rules and stop-words. The information of one index never acts on another index.
 
-This means that if you search for something that matches in the `description` of the document _A_ and in the `title` of the document _B_,
-the document _B_ will be considered better than the document _A_. You can read more about these rules [in the ranking section][1].
+This means, you can create synonyms for a `movie` index and different synonyms for a `clothes` index on the same MeiliSearch server.
```
Comment (curquiza) [inline guides/main_concepts/indexes.md]:
```suggestion
This means you can create synonyms for a `movie` index and different synonyms for a `clothes` index on the same MeiliSearch server.
```

File: guides/main_concepts/indexes.md [hunk 7]
```diff
@@ -1,152 +1,81 @@
...
-[1]: /guides/advanced_guides/ranking.md#ranking-rules
 
-<!-- TODO change doc link -->
-::: tip
-The **order of the document fields has a huge impact on the relevancy**. So please order fields from the most important to the less.
-:::
+## Index UID
 
-### Example
+The `uid` of an index is its **unique** identifier. It is the `:uid` parameter found on every `indexes/:uid` route.
```
Comment (curquiza) [inline guides/main_concepts/indexes.md]:
```suggestion
The `uid` of an index is its **unique** identifier. It is the `:index_uid` parameter found on every `indexes/:index_uid` route.
```

File: guides/main_concepts/indexes.md [hunk 8]
```diff
@@ -1,152 +1,81 @@
...
-If a document is added when there is already a document with the same ID in MeiliSearch, it will be updated.
-
-::: tip
-Documents identifiers are always converted into strings. Only strings and integers are valid identifiers.
-It means that it is, for example, forbidden to use arrays or objects as an identifier.
-:::
+## Document identifier
 
-## Indexed
+The document identifier is a <glossary word="field" /> in a document. This field is composed of an identifier <glossary word="attribute"/> name and it's unique value. All documents in a given index have the same identifier attribute, with each an unique value. The identifier's attribute name **must** be known by the index. There are [multiple ways to set your identifier](/guides/main_concepts/documents.md#setting-the-identifier).
```
Comment (curquiza) [inline guides/main_concepts/indexes.md]:
```suggestion
The document identifier is a <glossary word="field" /> in a document. This field is composed of an identifier <glossary word="attribute"/> name and its unique value. All documents in a given index have the same identifier attribute, with each a unique value. The identifier's attribute name **must** be known by the index. There are [multiple ways to set your identifier](/guides/main_concepts/documents.md#setting-the-identifier).
```

File: guides/main_concepts/indexes.md [hunk 9]
```diff
@@ -1,152 +1,81 @@
...
 
-It defines the fields that will be used to find the documents.
+[More information about the document identifier](/guides/main_concepts/documents.md#identifier)
 
-For example, an `url` field is not necessarily interesting for the search engine. So by omitting this property, the `url` will not be indexed but can still be returned inside the documents with the `displayed` property.
+## Relevancy rules
 
-## Displayed
+Each index has its own relevancy rules. By default, all index starts with the same <glossary word="ranking rules"/>  applied in the same order. Once you add your first document, from the order of the keys in this document, the index will be able to record which key is more important than another.
```
Comment (curquiza) [inline guides/main_concepts/indexes.md]:
```suggestion
Each index has its own relevancy rules. By default, all index starts with the same <glossary word="ranking rules"/> applied in the same order. Once you add your first document, from the order of the keys in this document, the index will be able to record which key is more important than another.
```

File: guides/main_concepts/indexes.md (hunk 5 above)
Comment (bidoubiwa) [reply guides/main_concepts/indexes.md]: Yes, bad wording

**Pull request merged**